packages/web-client/app/components/card-pay/create-merchant-workflow/merchant-customization/index.ts

```diff
@@ -24,12 +24,29 @@ export default class CardPayCreateMerchantWorkflowMerchantCustomizationComponent
   @tracked merchantIdValidationMessage = '';
   @tracked merchantBgColorValidationMessage = '';
 
+  constructor(owner: unknown, args: WorkflowCardComponentArgs) {
+    super(owner, args);
+
+    let { merchantName, merchantId, merchantBgColor } =
+      this.args.workflowSession.state;
+
+    if (merchantName && merchantId && merchantBgColor) {
```

Comment:
Maybe safer to use `isPresent` for all these args? Just to make sure they aren't empty strings